data/distro.yml

```diff
@@ -738,3 +738,32 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
+    
+- name: "Crystal Linux"
+  logo: "crystallinux.svg"
+  info: >
+    <ol class="distrotut">
+      <h2>Flatpak is installed by default on Crystal Linux, however if you didn't use jade_gui to install crystal or selected not to install it you can set flatpak up by using following commands<h3>
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> ame upg
+        </code></pre>
+        <p>Then install Flatpak:</p>
+          <pre><code>
+          <span class="unselectable">$</span> ame ins flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
```

Comment:
Is graphical installation of Flatpak apps (via GNOME Software, KDE Discover etc.) available in Crystal Linux?

Reply:
Yes, it installs gnome software by default on gnome

Reply:
Does the GNOME Software comes with Flatpak plugin preinstalled by default (or as a dependency of the `flatpak` package)?

Reply:
Yes, when installing gnome software flatpak works directly

Reply:
Ok, thanks!